i18n/en.json

```diff
@@ -1010,10 +1010,8 @@
   "admin.group_settings.group_row.edit": "Edit",
   "admin.group_settings.group_row.link_failed": "Link failed",
   "admin.group_settings.group_row.linked": "Linked",
-  "admin.group_settings.group_row.linking": "Linking",
```

Comment:
These shouldn't be removed, but that's probably happening because of what I mentioned earlier about the ternary statement being inside the `localizeMessage` arguments

Reply:
You are right. This is about ternary is being inside `localizeMessage` arguments